pywr/_model.pyx

```diff
@@ -140,6 +140,13 @@ class Model(object):
     def objectives(self):
         return NamedIterator(n for n in self.recorders if n.is_objective)
 
+    def is_feasible(self):
```

Comment:
worth calling this in `model.check`?

Reply:
The `model.check()` function was intended as sanity check of the model before simulation. You can't do this feasibility check before the simulation.

Reply:
ok

Reply:
I'll add a comment to the docstring.